pkg/domain/infra/abi/network.go

```diff
@@ -139,6 +139,7 @@ func (ic *ContainerEngine) NetworkRm(ctx context.Context, namesOrIds []string, o
 }
 
 func (ic *ContainerEngine) NetworkCreate(ctx context.Context, network types.Network) (*types.Network, error) {
+	// TODO: stop accepting "pasta" as value here by 5.x
```

Comment:
can you format this as `TODO (5.0): ...`, this will make greping it easier.

Reply:
Sure, I was wondering if there was a common format:

```
$ grep -rnI "TODO" | grep "[^.]5\.0"
libpod/networking_linux.go:749:// TODO (5.0): return the statistics per network interface
pkg/bindings/containers/checkpoint.go:86:		// TODO: remove ImportAchive with 5.0
pkg/specgen/specgen.go:488:	// TODO (5.0): change to pointer
cmd/podman/inspect/inspect.go:170:			// TODO (5.0): consider removing this to better match other commands.
```

...I see one is being established. I'll change it.

Reply:
Done.